docs/main/user-guides/replicated-storage-user-guide/rs-installation.md

```diff
@@ -232,20 +234,20 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
-## Minimum Worker Node Count
+### Minimum Worker Node Count
 
-  The minimum supported worker node count is three nodes. When using the synchronous replication feature (N-way mirroring), the number of worker nodes to which Mayastor is deployed should be no less than the desired replication factor.
+  The minimum supported worker node count is three nodes. When using the synchronous replication feature (N-way mirroring), the number of worker nodes to which Replicated Storage is deployed should be no less than the desired replication factor.
 
-## Transport Protocols
+### Transport Protocols
 
-  Mayastor supports the export and mounting of volumes over NVMe-oF TCP only. Worker node(s) on which a volume may be scheduled (to be mounted) must have the requisite initiator support installed and configured.
-  In order to reliably mount Mayastor volumes over NVMe-oF TCP, a worker node's kernel version must be 5.13 or later and the nvme-tcp kernel module must be loaded.
+  Replicated Storage supports the export and mounting of volumes over NVMe-oF TCP only. Worker node(s) on which a volume may be scheduled (to be mounted) must have the requisite initiator support installed and configured.
+  In order to reliably mount Replicated Storage volumes over NVMe-oF TCP, a worker node's kernel version must be 5.13 or later and the nvme-tcp kernel module must be loaded.
 
-## Preparing the Cluster
+### Preparing the Cluster
 
-### Verify/Enable Huge Page Support
+#### Verify/Enable Huge Page Support
 
-_2MiB-sized_  Huge Pages must be supported and enabled on the replicated engine storage nodes. A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the replicated engine pod on each node, which should be verified thus:
+_2MiB-sized_  Huge Pages must be supported and enabled on the Replicated Storage nodes. A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the Replicated Storage pod on each node, which should be verified thus:
```

Comment:
1. on the Storage nodes i.e. nodes where IO engine pods are deployed.
2. IO engine pod on each node, which...